privacy/models/privacy_activity.py

```diff
@@ -49,4 +49,4 @@ class PrivacyActivity(models.Model):
     @api.model
     def _default_controller_id(self):
         """By default it should be the current user's company."""
-        return self.env.user.company_id
+        return self.env.user.partner_id
```

Comment:
As you see in the docstring, this is incorrect. Instead:

```suggestion
        return self.env.user.company_id.partner_id
```